src/patternfly/components/Table/table.scss

```diff
@@ -902,9 +912,19 @@
 .pf-c-table__check {
   --pf-c-table--cell--FontSize: var(--pf-c-table__check--input--FontSize);
 
-  > input:not([disabled]) {
+  label:not([disabled]),
+  input:not([disabled]) {
     cursor: pointer;
   }
+
+  label {
+    position: absolute;
```

Comment:
Just a thought - would it make more sense to have the same padding as the table cell with negative margins equal to that padding? Then you wouldn't have to manually set the cell width.

Reply:
Yeah, I think that's a good point. Let me try that.